src/components/v5/common/CreateColonyWizard/validation.ts

```diff
@@ -115,7 +116,11 @@ export const tokenValidationSchema = object({
   .concat(createTokenValidationSchema);
 
 function isValidName(name: string) {
-  return name ? new RegExp(COLONY_NAME_REGEX).test(name) : true;
+  return name
```

Comment:
This prevents users from creating colonies with the same names as base routes, e.g. "404"

Reply:
I'm assuming it is possible to add a bunch of placeholder routes, to protect here.
I will put together a list.